frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -72,6 +72,13 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
                     ..Default::default()
                 },
             },
+            #[cfg(feature = "chrono")]
+            IrTypeDelegate::TimeList(ir) => Acc::distribute(Some(format!(
+                "final ans = Int64List(raw.length);
+                for (var i=0; i < raw.length; ++i) ans[i] = api2wire_Chrono_{}(raw[i]);
+                return api2wire_int_64_list(ans);",
+                ir.safe_ident()
```

Comment:
uber-nit: is it possible we somehow generalize this logic (List of anything)?

Reply:
I am just having a feeling that this code may be improveable (thus say "uber-nit" ;) ). Yes I think delegating to Int64List would be great if possible. Then, later if someone wants another similar thing (say "Apple" that is 32bit representable) they can use the same mechanism)

Reply:
Going back to this, I'm realizing that most of the delegated types require a `*List` variant to be correctly handled, not only because of *which* type (byte buffer or vector?) to use. I'd love to be able to write something that handles most cases in the most general way possible and make it easier to add implementations (or specializations) with minimal boilerplate, but current preoccupations mean that `TimeList` will have to suffice for now.

Reply:
Sure, so if you feel the current PR is ready, I will merge it :)